### ISSUE TITLE
Instructions on file setup and application configuration

### ISSUE THREAD

**Issue by Penecruz** [flying-huckleberry/SLbot#5]
Hi Just wondering if you have more specific information on installation of the application server side? The bot creation etc. are easily managed via discord support, just how to configure the app to run on the server, the integration with Slmod, and the best folder location.

**Comment by techdr1ve** [comment]
Agreed, would be awesome to get this information!

**Comment by techdr1ve** [comment]
@Penecruz 
So I looked into this, the way to get it to work is you have to get SLSC and S3 (both from Banjo who contribute to this GIT project as well) running in the background of the server, then use this SLbot for it to work. Now I haven't completely gotten everything to work here yet. So far that's what it seems like it takes to get this all to run.

SLSC
https://github.com/Bango1999/SLSC

S3
https://github.com/Bango1999/S3

**Comment by techdr1ve** [comment]
@Penecruz Could you help me how to get the discord portion of this to work? 
I'll help you get the rest of this running ( just can't get the final SLbot to work)

**Comment by Penecruz** [comment]
> @Penecruz Could you help me how to get the discord portion of this to work?
> I'll help you get the rest of this running ( just can't get the final SLbot to work)

Yes I have Hypeman outputting via a discord bot as well as DDC2 for server control via discord. The Discord side is easy it's the other side that is hard. Happy to help with the discord side side.

**Comment by techdr1ve** [comment]
Nice, i'll try this 'Hypeman' you talk of and the 'DDC2' for server control via discord, I was unaware of that.
I'll report back and see how that goes, in the meantime i'll try to get my buddy to help me for the SLbot and if that doesn't work i'll additionally try to find a way to contact the developer chasing this discord location somehow to help us.

**Comment by flying-huckleberry** [comment]
@Penecruz @ucnanoman apologies for the very late reply.

SLbot is unfinished and the readme is lacking in specifics, apologies about that.
SLbot integrates the S3 repo from Bango1999 with a discord bot script. The SLSC repo (also from Bango) that you mentioned before is still required to run independently.

So if you are hosting more than 1 DCS server, you would run 1 instance of SLSC for each server you're hosting, and then you only have to run 1 instance of SLbot, with the Web Server & Discord Bot checkboxes flipped on.  All of your SLSC instances need to be configured with tokens you setup in config.js, and they will feed their independent server data to SLbot, which will serve that data in web tables and query it via discord bot commands/web API interface.

If you have further questions about how to get this working, I'm happy to help answer them.

This repo hasn't been maintained for a few years, so some updates on my part might be required at this point, idk.
